### PR TITLE
W-15898668-check-logs-duke

### PR DESCRIPTION
**Pull request by dukesphere**
Update troubleshoot-missing-features.adoc

### CODE REVIEW

File: modules/ROOT/pages/troubleshooting.adoc [hunk 1]
```diff
@@ -20,9 +20,18 @@ For other issues and workarounds, see xref:acb-release-notes.adoc#known-issues[K
...
 For tasks that support the troubleshooting process, see:
 
+* <<check-logs>>
 * <<export-troubleshooting-info>>
 * <<manage-cloud>>
 
+[[check-logs]]
+== Check for Problems and Issues
+
+When an issue occurs, the first step is to check for problems and issues. The *Problems* tab lists warnings and error messages. The *Output* panel provides build details and access to logs that can help you understand underlying issues.
```
Comment (dukesphere) [inline modules/ROOT/pages/troubleshooting.adoc]:
```suggestion
When an issue that you are unable to resolve occurs, the first step is to check for problems and issues. The *Problems* tab lists warnings and error messages. The *Output* panel provides build details and access to logs that can help you understand underlying issues. 
```

Comment (dukesphere) [inline modules/ROOT/pages/troubleshooting.adoc]:
```suggestion
When an issue that you cannot resolve occurs, the first step is to check for problems and issues. The *Problems* tab lists warnings and error messages. The *Output* panel provides build details and access to logs. Error messages and logs list underlying issues that can help you understand what went wrong and what requires attention. 
```

File: modules/ROOT/pages/troubleshooting.adoc [hunk 2]
```diff
@@ -20,9 +20,18 @@ For other issues and workarounds, see xref:acb-release-notes.adoc#known-issues[K
...
 For tasks that support the troubleshooting process, see:
 
+* <<check-logs>>
 * <<export-troubleshooting-info>>
 * <<manage-cloud>>
 
+[[check-logs]]
+== Check for Problems and Issues
+
+When an issue that you cannot resolve occurs, the first step is to check for problems and issues. The *Problems* tab lists warnings and error messages. The *Output* panel provides build details and access to logs. Error messages and logs list underlying issues that can help you understand what went wrong and what requires attention.
```
Comment (dukesphere) [inline modules/ROOT/pages/troubleshooting.adoc]:
```suggestion
When an issue that you do not know how to resolve occurs, the first step is to check for and review problems, console output, and logs. 

The *Output* panel provides build details and access to logs. The *Problems* tab lists warnings and error messages. Error messages and logs list underlying issues that can help you understand what went wrong and what requires attention. 
```

File: modules/ROOT/pages/troubleshooting.adoc [hunk 3]
```diff
@@ -20,9 +20,20 @@ For other issues and workarounds, see xref:acb-release-notes.adoc#known-issues[K
...
 
+[[check-logs]]
+== Check for Problems and Issues
+
+When an issue that you do not know how to resolve occurs, the first step is to check for and review problems, console output, and logs. 
+
+The *Output* panel provides build details and access to logs. The *Problems* tab lists warnings and error messages. Error messages and logs list underlying issues that can help you understand what went wrong and what requires attention. 
+
+* To view project logs from the *Output* panel, see xref:start-open-panel.adoc[Output panel].
+* To view problems, open the *Problems* tab by pressing Press Cmd+Shift+m (Mac) or Ctrl+Shift+m (Windows).
```
Comment (dukesphere) [inline modules/ROOT/pages/troubleshooting.adoc]:
```suggestion
* To view problems, open the *Problems* tab by pressing Cmd+Shift+m (Mac) or Ctrl+Shift+m (Windows). 
```

File: modules/ROOT/pages/troubleshoot-missing-features.adoc [hunk 4]
```diff
@@ -5,6 +5,8 @@ include::reuse::partial$beta-banner.adoc[tag="anypoint-code-builder"]
 
 Some Anypoint Code Builder features can be disabled or hidden based on settings in the IDE. 
 
+For any features that are not working as expected, first xref:troubleshooting.adoc#check-logs[check project logs].
```
Comment (dukesphere) [inline modules/ROOT/pages/troubleshoot-missing-features.adoc]:
```suggestion
For any features that are not working as expected, first xref:troubleshooting.adoc#check-logs[check for problems and issues] .
```

File: modules/ROOT/pages/troubleshoot-missing-features.adoc [hunk 5]
```diff
@@ -5,6 +5,8 @@ include::reuse::partial$beta-banner.adoc[tag="anypoint-code-builder"]
 
 Some Anypoint Code Builder features can be disabled or hidden based on settings in the IDE. 
 
+For any features that are not working as expected, first xref:troubleshooting.adoc#check-logs[check for problems and issues] .
```
Comment (dukesphere) [inline modules/ROOT/pages/troubleshoot-missing-features.adoc]:
```suggestion
For any features that are not working as expected, first xref:troubleshooting.adoc#check-logs[check for problems and issues].
```

**Review by simoneegeib — DISMISSED** [review]

File: modules/ROOT/pages/troubleshooting.adoc [hunk 6]
```diff
@@ -20,9 +20,20 @@ For other issues and workarounds, see xref:acb-release-notes.adoc#known-issues[K
...
 * <<manage-cloud>>
 
+[[check-logs]]
+== Check for Problems and Issues
+
+When an issue that you do not know how to resolve occurs, the first step is to check for and review problems, console output, and logs. 
+
+The *Output* panel provides build details and access to logs. The *Problems* tab lists warnings and error messages. Error messages and logs list underlying issues that can help you understand what went wrong and what requires attention. 
+
+* To view project logs from the *Output* panel, see xref:start-open-panel.adoc[Output panel].
```
Comment (dukesphere) [inline modules/ROOT/pages/troubleshooting.adoc]:
```suggestion
* To view project logs from the *Output* panel, see xref:start-open-panel.adoc[].
```

File: modules/ROOT/pages/troubleshoot-missing-features.adoc [hunk 7]
```diff
@@ -5,6 +5,8 @@ include::reuse::partial$beta-banner.adoc[tag="anypoint-code-builder"]
 
 Some Anypoint Code Builder features can be disabled or hidden based on settings in the IDE. 
 
+For any features that are not working as expected, first xref:troubleshooting.adoc#check-logs[check for problems and issues].
```
Comment (chris-campbell9) [inline modules/ROOT/pages/troubleshoot-missing-features.adoc]:
```suggestion
For any features that aren't working as expected, first xref:troubleshooting.adoc#check-logs[check for problems and issues].
```
I believe our current guidance says to use contractions. But if I'm wrong, please disregard!

File: modules/ROOT/pages/troubleshooting.adoc [hunk 8]
```diff
@@ -20,9 +20,20 @@ For other issues and workarounds, see xref:acb-release-notes.adoc#known-issues[K
...
 For tasks that support the troubleshooting process, see:
 
+* <<check-logs>>
 * <<export-troubleshooting-info>>
 * <<manage-cloud>>
 
+[[check-logs]]
+== Check for Problems and Issues
+
+When an issue that you do not know how to resolve occurs, the first step is to check for and review problems, console output, and logs.
```
Comment (chris-campbell9) [inline modules/ROOT/pages/troubleshooting.adoc]:
```suggestion
When you don't know how to resolve an issue, the first step is to check for and review problems, console output, and logs. 
```
Suggestion to simplify a little. Also, I think we're supposed to use contractions now.

**Review by chris-campbell9 — APPROVED** [review]

**Pull request merged**